src/core/Entry.cpp

```diff
@@ -784,3 +789,20 @@ QString Entry::resolvePlaceholder(const QString& str) const
 
     return result;
 }
+
+QString Entry::resolveUrl(const QString& url) const
+{
+    QStringList parts = url.toLower().split("://");
```

Comment:
And also, passing to `QUrl` an url like `cmd://firefox www.google.com` result in:

```
qDebug() << uurl.scheme() << uurl.host() << uurl.url();
# "cmd" "" ""
```